Feature add

mrmuli opened the pull request.

Commit:
Add pypiglet and setup docopt file

Commit:
Merge pull request #9 from andela-jmuli/develop

Develop

Commit:
[Feature] Polish print room method

Commit:
[Feature] Add database class for Amity
Add connectdb method for db connection
Add commit_rooms method

Commit:
[Feature] Add commit_people method

Commit:
[Feature] Add save_state method

Commit:
bug fix for cyclic imports

Commit:
[Bug] fix cyclic imports

refactor code and add commit_person method to Person class, commit_room method to Room class

Commit:
add allocations model

Commit:
add session for database

Commit:
fix bad dictionary query

Commit:
Save State fix #1

Commit:
Merge pull request #10 from andela-jmuli/docopt-ui

Add pypiglet and setup docopt file

Commit:
docopt commands setup

Commit:
add command implementation

Commit:
command update

Commit:
PEP8

Commit:
reallocation bug fix

Commit:
command update

Commit:
[Feature] Add load_state method in database
Add load_rooms and load_people methods

Commit:
load occupant data

Commit:
new changes

njirap commented on App/models.py:
```diff
@@ -4,39 +4,36 @@
 
-
-engine = create_engine('sqlite:///amity.db', echo = False)
+# engine = create_engine('sqlite:///amity.db', echo = False)
```
commented code is bad for your health..

mrmuli replied:
noted

njirap commented on App/person.py:
```diff
@@ -124,45 +137,58 @@ def reallocate_person(self, person_id, room_name):
+            #         for occupant in Room.total_rooms[room]:
+            #             if person_id == occupant:
+            #                 Room.total_rooms[room].remove(person_id)
 
```
don't do commented code..

mrmuli replied:
noted

Commit:
[Chore] Correct circleci for checks

Commit:
modify add_person method account for unallocated

Commit:
[Feature]
modify print_allocations for neat  output
modify add_person for max allocations per room based on type

Commit:
[Chore]
remove comments
add test people

Commit:
chore -- add ignored files

Commit:
[Bug] system does not reallocate staff members to livingspaces

Commit:
[Chore] fix scrutinizer test setup

Commit:
[Bug] Fix person data save_state

Fix on commit people method -- staff members aren't allocated livingspaces
Add docstring and PEP8 formatting

Commit:
[Bug] print_unallocated fix

method prints different choices on unallocated people based on job type and room type

Commit:
[Bug] Fix reallocation method

method adds person to new room if not fully occupied

Commit:
[Bug] fix for room data get from database

Commit:
[bug] fix on reallocation

Commit:
write unallocated to optional file

Commit:
add integrity check

Commit:
[Feature] Add optional db requirement for save_state
Add save_state and load_state command implemetations

Commit:
[Feature] Add travis config file
Fix cyclic errors on test files
Add badge to README file

Commit:
fix travis

Commit:
add coverage

Commit:
coveralls
